inc/namespace.php

```diff
@@ -309,3 +348,188 @@ function get_random_weighted_element( array $weighted_values ) {
 		}
 	}
 }
+
+/**
+ * Fetch the automatically created demo audiences.
+ *
+ * @return array
+ */
+function get_demo_audiences() : array {
+	$existing = new WP_Query(
+		[
+			'post_type' => 'audience',
+			'meta_key' => '_altis_analytics_demo_data',
+			'posts_per_page' => 2,
+			'orderby' => 'post_name',
+			'order' => 'ASC',
+		]
+	);
+
+	return $existing->posts;
+}
+
+/**
+ * Create default audiences if none exist yet.
+ *
+ * @return void
+ */
+function maybe_create_audiences() {
+	$existing = get_demo_audiences();
+
+	// Audiences already exist.
+	if ( count( $existing ) === 2 ) {
+		return;
+	}
+
+	// Could happen, if it does we'll just delete it and recreate it.
+	if ( count( $existing ) === 1 ) {
+		foreach ( $existing as $audience ) {
+			wp_delete_post( $audience->ID, true );
+		}
```

Comment:
Any reason for a foreach here?
```suggestion
		wp_delete_post( $existing[0]->ID, true );
```

Reply:
Derp 🤦